codalab/lib/bundle_util.py

```diff
@@ -261,8 +271,6 @@ def recurse(old_bundle_uuid):
 
             prelude_items = []  # The prelude that we're building up
             for item in worksheet_info['items']:
-                just_added = False
```

Comment:
Why delete these?

Reply:
`just_added` hasn't been used in any condition check. Looks redundant to me.